<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
  margin: 0;
  width: 100vw;
  height: 100vh;
  background: #ecf0f3;
  display: flex;
  align-items: center;
  text-align: center;
  justify-content: center;
  place-items: center;
  overflow: hidden;
  font-family: poppins;
}

QFrame {
  position: relative;
  border-radius: 20px;
  box-sizing: border-box;
  background: #ecf0f3;
}
QLabel{
font: 14pt &quot;MS Shell Dlg 2&quot;;
}
QLabel, QLineEdit, #QPushButton {
  display: block;
  width: 100%;
  padding: 0;
  border: none;
  outline: none;
}


QLineEdit,QComboBox { 
text-align: left;
font: 14pt &quot;MS Shell Dlg 2&quot;;
  padding: 5px;
  padding-left: 8px;
  height: 32px;
  border-radius: 5px;
margin-top:8px;
}

QPushButton {
font: 13pt &quot;MS Shell Dlg 2&quot;;
  margin-top: 20px;
  background: #ff9900;
  height: 40px;
  border-radius: 5px;
  font-weight: 600;
color: #222;
}
#label {
font: 16pt &quot;MS Shell Dlg 2&quot;;
  margin-top: 10px;
  font-weight: 900;
  color: #ff9900;
  letter-spacing: 1px;
}


QPushButton:hover {
  background: #333;
color:#ff9900;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>360</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Admin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Foydalanuvchilar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Foydalanuvchi qo'shish</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Tovarlar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Tovar qo'shish</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Buyurtmalar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1" rowspan="2">
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Foydalanuvchi ma'lumotlari</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Fio</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Parol</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
              <item row="7" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>Tahrirlash</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QLineEdit" name="lineEdit_4"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Foydalanuvchilar</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QListWidget" name="listWidget">
             <property name="styleSheet">
              <string notr="true">QWidget { 
text-align: left;
font: 14pt &quot;MS Shell Dlg 2&quot;;
  padding: 5px;
  padding-left: 8px;
  height: 32px;
  border-radius: 5px;
margin-top:8px;
	background-color: rgb(203, 203, 203);
}
QListWidget{
	background-color: rgb(255, 254, 255);
}
</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>O'chirish</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Foydalanuvchi Qo'shish</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Fio</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_6"/>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_7">
                <property name="text">
                 <string>Qo'shish</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_7"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Parol</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Tovarlar</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Tovar ma'lumotlari</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Nomi:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_8"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Kategoriyasi</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>TV</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Laptop</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mobile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>aksesuar</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>maishiy</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_10"/>
              </item>
              <item row="9" column="0" colspan="2">
               <widget class="QLineEdit" name="lineEdit_12"/>
              </item>
              <item row="10" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_10">
                <property name="text">
                 <string>Tahrirlash</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Soni:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEdit_11"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Narxi:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QListWidget" name="listWidget_2">
             <property name="styleSheet">
              <string notr="true">QWidget { 
text-align: left;
font: 14pt &quot;MS Shell Dlg 2&quot;;
  padding: 5px;
  padding-left: 8px;
  height: 32px;
  border-radius: 5px;
margin-top:8px;
	background-color: rgb(203, 203, 203);
}
QListWidget{
	background-color: rgb(255, 254, 255);
}
</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>O'chirish</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_6">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Tovar Qo'shish</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Nomi:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_13"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Kategroriyasi:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Soni:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_15"/>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_11">
                <property name="text">
                 <string>Qo'shish</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Narxi:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEdit_16"/>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="comboBox_2">
                <item>
                 <property name="text">
                  <string>TV</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Laptop</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mobile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>aksesuar</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>maishiy</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QTableView" name="tableView">
             <property name="styleSheet">
              <string notr="true">QWidget { 
text-align: left;
font: 12pt &quot;MS Shell Dlg 2&quot;;
  padding: 5px;
  padding-left: 8px;
  height: 40px;
  border-radius: 5px;
margin-top:8px;
	background-color: rgb(203, 203, 203);
}
QTableView{
	background-color: rgb(255, 254, 255);
}
</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Buyurtmalar</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1238</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
